docs/reference/standard-library.rst

```diff
@@ -38,40 +38,77 @@ Int
 Sequences
 ============
 
-Required for `pluscal procedures <procedure>`.
+Required for `pluscal procedures <procedure>`. A sequence is like a list in any other language. You write it like `<<a, b, c>>`, and the elements can be any other values (including other sequences).
 
 Seq(set)
-  The set of all sequences where the values are all elements of ``set``. |noenumerate| 
+  The set of all sequences where the values are all elements of ``set``. As the output is computed in a lazy way, this is not memory/cpu intensive. |noenumerate|
```

Comment:
The output isn't computed in a lazy way, TLC just has special code to check if you're testing membership. If you try to select any values from the sequence then it will fail (because it's nonenumerable)

Reply:
I tried to rephrase it, let me know if reads better or if I should just remove this sentence. I mentioned it because the output of Seq({a, b}) in TLC repl is Seq({a, b}), and thought it would be helpful to understand what's going on (of course, I haven't checked the code but I just assumed it was involving some lazy computation)